build-image/first-boot.bash

```diff
@@ -156,18 +156,22 @@ ln -sfn /opt/openhabian/openhabian-setup.sh /usr/local/bin/openhabian-config
 echo "$(timestamp) [openHABian] Executing 'openhabian-setup.sh unattended'... "
 if (/bin/bash /opt/openhabian/openhabian-setup.sh unattended); then
 #if (/bin/bash /opt/openhabian/openhabian-setup.sh unattended_debug); then
-  systemctl start openhab2.service
+  service openhab2 start
   rm -f /opt/openHABian-install-inprogress
   touch /opt/openHABian-install-successful
 else
   fail_inprogress
 fi
 echo "$(timestamp) [openHABian] Execution of 'openhabian-setup.sh unattended' completed."
 
-echo -n "$(timestamp) [openHABian] Waiting for openHAB to become ready... "
-until wget -S --spider http://localhost:8080 2>&1 | grep -q 'HTTP/1.1 200 OK'; do
+echo -n "$(timestamp) [openHABian] Waiting for openHAB to become ready..."
+i=0
+until wget -S --spider -t 3 --waitretry=4 http://localhost:8080 2>&1 | grep -q 'HTTP/1.1 200 OK' || [ $i -ge 10 ]; do
```

Comment:
I like this part 👍 We have plenty of places thought out the code which could use this pattern.
Could you make this to a separate function in the helper section?

Maybe something like this?
```
untilRdy(String checkCommand, Int tries) {
...
} 
```

Reply:
Will do midterm but for now I mustn' lose the overview so won't.
Still WIP with this inside Docker , the dashboard won't come up.